app/src/openxr/cpp/DeviceDelegateOpenXR.cpp

```diff
@@ -868,6 +898,12 @@ DeviceDelegateOpenXR::ProcessEvents() {
         m.reorientRequested = true;
         VRB_DEBUG("OpenXR: reference space changed. User recentered the view?");
         break;
+      case XR_TYPE_EVENT_DATA_PASSTHROUGH_STATE_CHANGED_FB: {
+        const auto &event =
+                *reinterpret_cast<const XrEventDataPassthroughStateChangedFB *>(ev);
```

Comment:
Nit: single line please. Also it's a bit weird to dereference a reinterpret cast. Why don't you just pass `*event` to the `HandlePassthroughEvent` instead?

Reply:
This is what they do for all events in `ProcessEvents()` :shrug:

Reply:
OK, then leave it as is